test/e2e/network/scale/localrun/ingress_scale.go

```diff
@@ -19,6 +19,7 @@ package main
 import (
 	"flag"
 	"fmt"
+	"k8s.io/kubernetes/test/e2e/network"
```

Comment:
In this case (used at multiple test packages), we don't need to move the functions.

Reply:
@oomichi got it, let me close this pull request